test/machineslib.py

```diff
@@ -277,6 +277,10 @@ def setUp(self):
         # Ignoring just to unbreak tests for now
         self.allow_journal_messages("Failed to get COMM: No such process")
 
+        # FIXME: Testing on Arch Linux fails randomly with networkmanager time outs while the test passes.
```

Comment:
This commit message has a "networmanager" typo. Please fix it to unbreak commit log searches.